tslint.json

```diff
@@ -12,6 +12,9 @@
       ],
       "ban-types": [
         false
+      ],
+      "no-console": [
+        false
```

Comment:
probably nice to set a warning here, not disabling here. `ban-types` should warn on using `any`.